### PR TITLE
Update base permissions docs to explicitly state changes impact new & existing customers.

### LINKED ISSUE #24144: Explicitly state that changes to base permissions impact both new and existing members.
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/enterprise-server@3.8/organizations/managing-user-access-to-your-organizations-repositories/setting-base-permissions-for-an-organization

### What part(s) of the article would you like to see updated?

Add a note explicitly stating changes will occur on both new and existing members. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by adamrr724**
### Why:

Closes https://github.com/github/docs/issues/24144

### What's being changed (if available, include any code snippets, screenshots, or gifs):

Adding a note that states base permissions changes take effect on both new and existing customers. 

### Check off the following:

- [ ] I have reviewed my changes in staging (look for the "Automatically generated comment" and click the links in the "Preview" column to view your latest changes).
- [X] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### CODE REVIEW

**Comment by cmwilson21** [comment]
👋 @adamrr724 Thanks for opening an issue and submitting a PR! I edited your PR slightly, removing the word `ISSUE` after the `Closes` prompt to link the PR to the issue. 👍 

I'll get this triaged for review! :zap:

File: content/organizations/managing-user-access-to-your-organizations-repositories/setting-base-permissions-for-an-organization.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ If someone with admin access to an organization's repository grants a member a h
 If you've created a custom repository role with an inherited role that is lower access than your organization's base permissions, any members assigned to that role will default to the organization's base permissions rather than the inherited role. For more information, see "[AUTOTITLE](/organizations/managing-peoples-access-to-your-organization-with-roles/managing-custom-repository-roles-for-an-organization)."
 {% endif %}
 
+{% note %}
+Note: All changes to base permissions will take effect on both new and existing members. 
+{% endnote %}
```
Comment (sophietheking) [inline content/organizations/managing-user-access-to-your-organizations-repositories/setting-base-permissions-for-an-organization.md:33]:
```suggestion
{% note %}

**Note:** All changes to base permissions will affect both new and existing members. 

{% endnote %}
```

I edited the formatting and wording slightly. 😄

**Review by sophietheking — DISMISSED** [review]
@adamrr724 👋 thanks for opening this PR! I left a suggestion to update the note slightly.

Please feel free to `@mention` me here to get this merged!

**Comment by adamrr724** [comment]
Hi @sophietheking! All committed and ready for merge. Thank you so much!

**Review by sophietheking — APPROVED** [review]

**Pull request merged**